lib/hammer_cli_csv/content_views.rb

```diff
@@ -9,6 +9,9 @@ class ContentViewsCommand < BaseCommand
       command_name 'content-views'
       desc         'import or export content-views'
 
+      option %w(--no-publish), :flag, _('Skip publishing content views on import')
+      option %w(--no-promote), :flag, _('Skip promoting content views on import')
```

Comment:
Perhaps there should be a small bit of validation here; if `--export` is given, these options should be disallowed. Additionally, you might consider having a validation for `--no-publish` requires `--no-promote`. If you negate the options, you'd require `--publish` if you have `--promote`. Of course, you could also change the option description to be `option %w(--promote), :flag, _('Publish and promote content views on import')`; maybe that's the better solution.